packages/@expo/cli/src/run/ios/options/promptDevice.ts

```diff
@@ -28,7 +28,7 @@ export function formatDeviceChoice(item: AnyDevice): { title: string; value: str
         : '';
   const format = isActive ? chalk.bold : (text: string) => text;
   return {
-    title: `${symbol}${format(item.name)}${
+    title: `${symbol}${format(item.name)}${isConnected ? ` - ${item.modelName}` : ''}${
```

Comment:
Not very aesthetic. Maybe only show it if there are multiple devices with the same name.

Reply:
Definitely agree. Since these are my first changes in this repository, let me know if the most recent changes comply with Expo code standards.